README.md

```diff
@@ -55,6 +55,11 @@ console.log(errors.AccessDenied);
 
 ```
 
+## Build Status
+
+![Public Build Status][badgepub]
+![Private Build Status][badgepriv]
+
```

Comment:
+1 for adding badges, but these badges do not seem relevant:

```
[badgepub]: https://circleci.com/gh/scality/Arsenal.svg?style=svg
[badgepriv]: http://ci.ironmann.io/gh/scality/Arsenal.svg?style=svg&circle-token=c3d2570682cba6763a97ea0bc87521941413d75c
```

Either do not add the badges, or actually fix them, based on gha and codecov.
Not sure what the "private build status" is.